assembl/templates/views/adminTimelineEvent.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="subpanel-body bx">
+  <div>
+    <div class="admin-msg-elm">
+      <div class="control-group">
+        <label for="start_date" class="control-label">{{ gettext('Start date') }}</label>
+        <div class="controls">
+          <input class="js_start_date input-medium" id="start_date" name="start_date" type="datetime-local" value="<%= event.getStartNoTZ() %>"></input>
```

Comment:
Okay, so our agreement on this was to take the time to comment what time indicates, and be more explicit about the timezone stored. Furthermore, comment in code made for front-end to be aware that the incoming datetime from backend is in UTC.